recipe/meta.yaml

```diff
@@ -0,0 +1,177 @@
+{% set name = "pykx" %}
+{% set version = "2.1" %}
+# this switch disables all upstream test because of a time out issue on some platforms that still
+# needs investigation
+{% set upstream_tests_are_enabled = False %}
+# this switch should be enabled if a license can be used to run the tests below (see base64 license)
+{% set kx_license_available = False %}
+# this switch should be enabled if kdb can be downloaded as a test dependency
+{% set install_kdb_as_test_dep = False %}
+# # this switch enables experimental win-64 build depending on vs2010_runtime
+{% set winbuild_enabled = False %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/KxSystems/pykx
+  git_rev: release/{{ version }}
```

Comment:
Could we pin to a commit instead of a branch?

Reply:
I could pin to the merge commit. That's when the branch was merged back to main, though.

Reply:
Branches can change, commits can't (or at least it's very hard to change a commit and re-apply it while keeping the same checksum and still getting a working software). So we are doing this for both a reproducibility reason and also for security.

Reply:
And even if we used their branch instead of a commit, their branch only nsays 2.1 while they technically published multiple bugfix releases on PyPI. See https://pypi.org/project/pykx/#history. They have published 2.1.0, 2.1.1 and 2.1.2. This means that in theory, we shouldn't even trust their GH repo since they don't tag nor create releases.